README.md

```
@@ -14,6 +14,26 @@
 - Flexible output formats (TSV and JSON)
 - Compatible with Neovim plugin [Perec](https://github.com/7sedam7/perec.nvim)
 
+## Performance
+
+Some people have said that they'd prefer if frontmatter info would be stored in some kind of database for performance reasons.
+I did some benchmarking, and on base Mac mini M4, Krafna can query ~2500 files within ~100ms.
+While fetching the files and parsing them takes about 97% of the time and I have ideas for optimisation (cache parsed files, and on runs parse only the ones that were modified after cache creation), 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)